src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -152,7 +151,7 @@ public async Task GetConfig_NoSupportedActiveRules_ReturnsNull()
         }
 
         [TestMethod]
-        public async Task GetConfig_ReturnsCorrectFilePath()
+        public void GetConfig_ReturnsCorrectRuleset()
```

Comment:
Presumably these could be actually be `async`, if the issue before was a merge conflict?

Reply:
yep, I can revert it back